plugins/notion/src/blocksToHTML.ts

```diff
@@ -94,6 +94,24 @@ export function blocksToHtml(blocks: BlockObjectResponse[]) {
             case "code":
                 htmlContent += `<pre><code class="language-${block.code.language.replace(" ", "-")}">${richTextToHTML(block.code.rich_text)}</code></pre>`
                 break
+            case "quote":
+                htmlContent += `<blockquote>${richTextToHTML(block.quote.rich_text)}</blockquote>`
+                break
+            case "video":
+                if (block.video.type === "external") {
+                    const url = block.video.external.url
+                    if (url && (url.includes("youtube.com") || url.includes("youtu.be"))) {
+                        const videoProps = {
+                            url: { type: "string", value: url },
+                            play: { type: "enum", value: "Off" },
+                            shouldMute: { type: "boolean", value: true },
+                        }
+                        htmlContent += `<template data-module-identifier="module:NEd4VmDdsxM3StIUbddO/9rhBPUZttCbLCWqJEL42/YouTube.js:Youtube" data-module-props='${JSON.stringify(
+                            videoProps
+                        )}'></template>`
+                    }
+                }
+                break
```

Comment:
For external videos (Youtube and Vimeo), we use iframes directly

- **YouTube**:
```html
<iframe src="https://www.youtube.com/embed/<videoId>?controls=0&autoplay=1&loop=0&mute=1"></iframe>
```
- **Vimeo**:
```html
<iframe src="https://player.vimeo.com/video/<videoId>?controls=0&autoplay=1&loop=0&muted=1"></iframe>
```

Reply:
Updated to use an iframe with the /embed URL for YouTube videos. Vimeo videos aren't supported in formatted text in Framer.

Reply:
We do have support for Vimeo, it's not visible in the UI, but if you paste a Vimeo link it will embed it.